### PR TITLE
#910  Fix excluding products from price

### LINKED ISSUE #910: Fix products exclude from price
Production config contains `7387` product excluded, but price contains the product anyway `<offer id="7387" available="true">`

### PR DESCRIPTION
**Pull request by duker33**
Closes #910

### CODE REVIEW

File: shopelectro/models.py [hunk 1]
```diff
@@ -82,6 +82,8 @@ class Product(
         verbose_name=_('tags'),
     )
 
+    # @todo #910:30m  Explore and doc what is vendor_code.
+    #  And render it as some hidden option on the product page.
```
Comment (artemiy312) [inline shopelectro/models.py]: >render it as some hidden option

What did you mean? Should we remove this from product page?

File: shopelectro/models.py [hunk 2]
```diff
@@ -82,6 +82,8 @@ class Product(
         verbose_name=_('tags'),
     )
 
+    # @todo #910:30m  Explore and doc what is vendor_code.
```
Comment (artemiy312) [inline shopelectro/models.py]: I still remember this story :) The root of evil here 1C integration

File: shopelectro/models.py (hunk 1 above)
Comment (duker33) [reply shopelectro/models.py]: i'll just rm this string

File: shopelectro/models.py (hunk 2 above)
Comment (duker33) [reply shopelectro/models.py]: @artemiy312 , then it's task for you =)

**Pull request merged**